client/src/components/Donation/stripe-card-form.tsx

```diff
@@ -136,7 +136,10 @@ const StripeCardForm = ({
   };
 
   return (
-    <Form className='donation-form' onSubmit={handleSubmit}>
+    <form
+      className='donation-form'
+      onSubmit={event => void handleSubmit(event)}
```

Comment:
The other changes should be "affecting" the styles, except this one, it affects the functionality, as well

Although I think, that it is the same functionality, it doesn't hurt to check it.